### ISSUE TITLE
Svelte Sapper ReferenceError: window is not defined

### ISSUE THREAD

**Issue by rohanharikr** [andrelmlins/svelte-internet-connection#1]
Installed as dev dependency.

```
500

window is not defined

ReferenceError: window is not defined
    at /Users/rohanharikumar/Documents/fullstack/client/node_modules/svelte-internet-connection/src/lib/InternetConnection.svelte:21:10
    at run (/Users/rohanharikumar/Documents/fullstack/client/node_modules/svelte/internal/index.mjs:18:13)
    at Array.forEach (<anonymous>)
    at run_all (/Users/rohanharikumar/Documents/fullstack/client/node_modules/svelte/internal/index.mjs:24:15)
    at Object.render (/Users/rohanharikumar/Documents/fullstack/client/node_modules/svelte/internal/index.mjs:1350:19)
    at /Users/rohanharikumar/Documents/fullstack/client/src/node_modules/@sapper/server.mjs:4676:54
    at Generator.next (<anonymous>)
    at fulfilled (/Users/rohanharikumar/Documents/fullstack/client/src/node_modules/@sapper/server.mjs:128:61)
```

**Comment by andrelmlins** [comment]
Hi there, I'm analyzing.

**Comment by andrelmlins** [comment]
Thanks for opening the incident, corrected in version 0.1.2

**Comment by rohanharikr** [comment]
@andrelmlins Neat! Thank you so much
